video/acceptance/video_samples_test.rb

```diff
@@ -13,121 +13,103 @@
 # limitations under the License.
 
 require_relative "../video_samples"
-require "rspec"
-require "tempfile"
+require "minitest/autorun"
+require "minitest/focus"
+require "minitest/hooks/default"
 require "net/http"
+require "securerandom"
+require "tempfile"
 require "uri"
 
 describe "Google Cloud Video API sample" do
-  before do
+  before :all do
     @labels_file        = "cloud-samples-data/video/cat.mp4"
     @shots_file         = "cloud-samples-data/video/gbikes_dinosaur.mp4"
     @safe_search_file   = "cloud-samples-data/video/pizza.mp4"
     @transcription_file = "cloud-samples-data/video/googlework_short.mp4"
   end
 
   it "can analyze labels from a gcs file" do
-    expect {
+    assert_output(/Label description: animal/) do
       analyze_labels_gcs path: "gs://#{@labels_file}"
-    }.to output(
-      /Label description: animal/
-    ).to_stdout
+    end
   end
 
   it "can analyze labels from a local file" do
     begin
-      local_tempfile = Tempfile.new "temp_video"
+      local_tempfile = Tempfile.new "temp_video#{SecureRandom.hex}"
```

Comment:
You do not need the `SecureRandom`. The `Tempfile` class automatically generates a unique name in the temporary directory.

Reply:
fixed